galley/pkg/server/server.go

```diff
@@ -229,6 +238,9 @@ func (s *Server) Run() {
 			scope.Fatalf("Galley Server unexpectedly terminated: %v", err)
 		}
 	}()
+	if s.callOut != nil {
```

Comment:
This needs to be included in s.serveWG as well

Reply:
Done